lib/logstash/inputs/syslog.rb

```diff
@@ -165,7 +184,7 @@ def tcp_listener(output_queue)
 
   public
   def teardown
-    @shutdown_requested = true
+    @shutdown_requested.update{|v| true}
```

Comment:
why? when you call a method:
```ruby
do_something(with_a_parameter)
```
we don't add extra spaces?

what is the difference? (I am being a dick here I know, but this is actually a styling issue I am struggling with - please help me find peace & balance with it ;)